euth/phases/contents.py

```diff
@@ -0,0 +1,30 @@
+from django.apps import apps
+
+class PhaseContent():
+    @property
+    def identifier(self):
+        return '{}:{:03d}:{}'.format(self.app, self.weight, self.phase)
+
+    def __str__(self):
+        return '{}({}:{})'.format(self.__class__.__name__, self.app, self.phase)
+
+
+class PhaseContents():
+    def __init__(self):
+        self._registry = {}
```

Comment:
If you don't do anything with `self._registry` in `__init__()`  you can simply set the default by defining an attribute:
```python
class PhaseContents():
    _registry = {}

    def __getitem__(self, identifier):
    ...
```